types/plotly.js/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for plotly.js 2.12
+// Type definitions for plotly.js 2.24
```

Comment:
I do not see any reason to update the version to 2.24. Looking at [plotly.js/src/traces/bar/attributes.js](https://github.com/plotly/plotly.js/blob/96c28159c06a3af3fe551175db89599894efbf82/src/traces/bar/attributes.js#L74), git blame on the properties of `textposition`, `insidetextanchor`, and `textangle` shows all of them to be at least 4 years old.

Reply:
Recommend reverting this change.